CONTRIBUTING.md

```diff
@@ -160,10 +160,12 @@ git submodule update --init --remote --recursive
 conda env create --name cudf_dev --file conda/environments/cudf_dev_cuda10.0.yml
 # activate the environment
 source activate cudf_dev
+#If conda is already in PATH, perform `conda activate cudf_dev`
```

Comment:
```suggestion
```

Reply:
Do I resubmit with this, or leave it at it ?